manager_test_helper.go

```diff
@@ -354,13 +372,15 @@ func TestHelperCreateGetDelete(s Manager) func(t *testing.T) {
 			found := map[string]int{}
 			for _, got := range pols {
 				for _, expect := range TestManagerPolicies {
-					if got.GetID() == expect.GetID() {
+					//This is a modified equality check
+					if got.GetID() == expect.GetID() && reflect.DeepEqual(got.GetResources(), expect.GetResources()) && reflect.DeepEqual(got.GetActions(), expect.GetActions()) {
 						found[got.GetID()]++
 					}
 				}
 			}
 
 			for _, f := range found {
+				//This assert is supposed to fail
```

Comment:
That's not true, found is a `map[string]int{}` which has for every policy id a value of 1 so this can't fail. A better test would be something like this (pseudocode):

```go
 			found := map[string]int{}
 			for _, got := range pols {
 				for _, expect := range TestManagerPolicies {
					if got.GetID() == expect.GetID() {
                        DeepEquals(expect, got)
 						found[got.GetID()]++
 					}
 				}
 			}
```